internal/validate/kube/kube.go

```diff
@@ -387,3 +434,106 @@ func Connections(ctx context.Context, config *Config) ([]validate.Result, error)
 
 	return results, nil
 }
+
```

Comment:
EKS functions from here down should be moved to `eks.go` file as well

Reply:
Moved!